benchmarks/lasso/solvers/glmnet.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+
+from benchopt.base import BaseSolver
+from benchopt.util import safe_import
+
+
+with safe_import() as solver_import:
+    import rpy2.robjects.packages as rpackages
+    from rpy2 import robjects
+    from rpy2.robjects import numpy2ri
+
+
+class Solver(BaseSolver):
+    name = "glmnet"
+
+    install_cmd = 'python'
```

Comment:
Yep this is `bash` and you need to furnish the name of a bash script file stored in `install_scripts`.
The command that is called is `bash install_scripts/{install_script} {env}` and is run with the correct environment activated I think.

This looks quite sub-optimal I agree :)